### PR TITLE
Do not include double quotes in strings

### PR DESCRIPTION
**Pull request by jvmakine**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,4 +2,13 @@ module github.com/alecthomas/protoparser
 
 go 1.17
 
-require github.com/alecthomas/participle/v2 v2.0.0-alpha7
+require (
+	github.com/alecthomas/participle/v2 v2.0.0-alpha7
+	github.com/stretchr/testify v1.4.0
```
Comment (alecthomas) [inline go.mod]: I was hoping to exclude testify in this package, just to keep it super lean?

File: parser.go [hunk 2]
```diff
@@ -20,7 +20,7 @@ type Proto struct {
 type Entry struct {
 	Pos lexer.Position
 
-	Syntax  string   `  "syntax" "=" @String`
+	Syntax  string   ` "syntax" "=" @String`
```
Comment (alecthomas) [inline parser.go]: ?

**Review by alecthomas — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (jvmakine) [reply go.mod]: 👍  Sounds good, will remove

File: parser.go (hunk 2 above)
Comment (jvmakine) [reply parser.go]: Oops

**Pull request merged**